spec/requests/providers/check_benefits_spec.rb

```diff
@@ -28,7 +28,7 @@
       let!(:benefit_check_result) { create :benefit_check_result, legal_aid_application: application }
 
       it 'does not generate a new one' do
-        expect_any_instance_of(BenefitCheckService).not_to receive(:call).and_call_original
+        expect(BenefitCheckService).not_to receive(:call).and_call_original
```

Comment:
I know it's not your code but I don't think `.and_call_original` is needed if `BenefitCheckService` is not supposed to receive `call` anyway